src/index.ts

```diff
@@ -16,31 +16,24 @@ import CONSTANTS from "./constants";
 
 export {
   // From event
-  CloudEvent,
+  CloudEvent, Version,
```

Comment:
I would keep identitiers on separate lines. Here and elsewhere.

Reply:
These formatting changes were auto applied by vscode. Perhaps we should have a lint rule?